src/concrete_convex_hull.jl

```diff
@@ -6,9 +6,58 @@ function default_convex_hull_algorithm(points)
     end
 end
 
+"""
+    function convex_hull(X::LazySet{N}, Y::LazySet{N}; [algorithm]=nothing,
+                         [backend]=nothing, [solver]=nothing) where {N<:Real}
+
+Compute the convex hull of the given convex sets.
+
+### Input
+
+- `X`         -- convex set
+- `Y`         -- convex set
+- `algorithm` -- (optional, default: `nothing`) the convex-hull algorithm
+- `backend`   -- (optional, default: `nothing`) backend for polyhedral
+                 computations (used for higher-dimensional sets)
+- `solver`    -- (optional, default: `nothing`) the linear-programming solver
+                 used in the backend
+
+### Output
+
+If the input sets are one-dimensional, the result is an `Interval`.
+If the input sets are two-dimensional, the result is a `VPolygon`.
+Otherwise the result is a `VPolytope`.
+
+### Algorithm
+
+One-dimensional sets are resolved by using `overapproximate` with an `Interval`
+(which is exact).
+For higher-dimensional sets, we compute the vertices of both `X` and `Y` using
+`vertices_list` and then compute the convex hull of the union of those vertices.
+"""
+function convex_hull(X::LazySet{N}, Y::LazySet{N};
+                     algorithm=nothing,
+                     backend=nothing,
+                     solver=nothing
+                    ) where {N<:Real}
+    n = dim(X)
+    @assert n == dim(Y) "the convex hull requires two sets of the same " *
+                        "dimension, but the sets had dimension $n and $(dim(Y))"
+    if n == 1
+        return overapproximate(ConvexHull(X, Y), Interval)
```

Comment:
There exists a special handling of the one-dimensional case. I would rather keep the line `chull` below, and add a case `if n == 1` to return an `Interval`.

Reply:
Hm, not sure. For lazy sets I would expect that the support function is much more efficient than `vertices_list`.

Reply:
Yes, it will be more efficient in general. But this ultimately depends on the implementation of overapproximate for given types. For example, it may as well return an interval that is strictly bigger than the concrete convex hull, and this is not the expected answer for `convex_hull`, that should be exact.

Reply:
Fine, I changed it and created #1638.